### PR TITLE
cross compile to scala 2.10 and 2.11

### PR DESCRIPTION
**Pull request by yanns**

### CODE REVIEW

**Comment by yanns** [comment]
@sphereio/sphere-team please review

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
  language: scala
  scala:
    - "2.10.4"
+   - "2.11.5"
```
Comment (schleichardt) [inline .travis.yml]: How does this affect our tests?

Comment (yanns) [inline .travis.yml]: @schleichardt the whole build chain is triggered two times: one time with Scala 2.10.4 and one time with Scala 2.11.5. See https://travis-ci.org/sphereio/sphere-scala-libs/builds/52184796 as an example.

**Comment by agourlay** [comment]
:clap:

**Comment by OlegIlyenko** [comment]
:+1: Nice! Thanks for doing this!

**Pull request merged**